### ISSUE TITLE
Both Lists Superimposed with "0 selected" in Chromium-based Mobile Browsers

### ISSUE THREAD

**Issue by rpowell6** [istvan-ujjmeszaros/bootstrap-duallistbox#169]
I am having an issue wherein both the selected and non-selected lists are superimposed with a "0 selected" message in Chromium-based mobile browsers.  The lists are not visible but are selectable by tapping on the down arrow on the respective controls.  Please note, this behavior is not observed in Firefox.

I have attached a screen shot for illustration purposes.  I have validated that the non-selected list is refreshed and options are selectable via the down arrow on the right side of the control.  Likewise, after selecting and moving an option, the option is visible in the selected list by tapping on the down arrow on the control.

I am new to "Bootstrap Dual Listbox" and I would like to continue using it.  Any assistance would be greatly appreciated.

![Screenshot_DualListBox_Chrome_2](https://user-images.githubusercontent.com/54381234/81875307-99923780-9545-11ea-8723-895a6565174a.jpg)

**Comment by sunil1979** [comment]
Is there a fix to this yet?

**Comment by rpowell6** [comment]
No, I am still waiting for a response.  I am to the point of "pulling the plug" on this plug-in unless something can be provided in the way of a solution.

**Comment by manukieli** [comment]
Hello, I've the same issue on mobile devices.
I'm searching an alternative, do you know one??

**Comment by rpowell6** [comment]
Hello to you as well.

No, I haven’t found any sort of open source alternative.  I’ve tried a couple of alternatives but they were significantly inferior to the bootstrap-duallistbox plug-in.  There may be plug-ins for purchase that are more reliable but I haven’t taken the time to search.  In the meantime, I’m telling my users to use Firefox.



 [cid:gw_6a5cf058-0b5b-46eb-8ba1-d614a42b779c.PNG]      Randall Powell
System Integrator

GW Communication Solutions, LLC
801 Hammond Street, Suite 350, Coppell, TX 75019

www.unitekglobalservices.com<http://www.unitekglobalservices.com/>

Important: Neither Unitek Global Services, Inc. nor its affiliates (together, “Unitek”), wish to receive, on a non-solicited basis, directly or indirectly, any non-public information related to any public issuer. Additionally, this e-mail may contain information that is confidential, privileged or otherwise protected from disclosure. If you are not an intended recipient of this e-mail, do not duplicate or redistribute it by any means. Please delete it and any attachments and notify the sender that you have received it in error. Unintended recipients are prohibited from taking action on the basis of information in this e-mail. Unitek reserves the right, to the extent and under circumstances permitted by applicable law, to review, retain, monitor and intercept e-mail messages to and from its systems. Unitek may also retain and reproduce electronic messages for state, federal, or other regulatory agencies as required by applicable law.

**Comment by rpowell6** [comment]
I tried a different jQuery/JavaScript plug-in (jQuery Dualselect) but got exactly the same result.  The issue appears to be with Chromium and not the “duallistbox” plug-in.


Best regards,

 [cid:gw_6a5cf058-0b5b-46eb-8ba1-d614a42b779c.PNG]      Randall Powell
System Integrator

GW Communication Solutions, LLC
801 Hammond Street, Suite 350, Coppell, TX 75019

www.unitekglobalservices.com<http://www.unitekglobalservices.com/>

Important: Neither Unitek Global Services, Inc. nor its affiliates (together, “Unitek”), wish to receive, on a non-solicited basis, directly or indirectly, any non-public information related to any public issuer. Additionally, this e-mail may contain information that is confidential, privileged or otherwise protected from disclosure. If you are not an intended recipient of this e-mail, do not duplicate or redistribute it by any means. Please delete it and any attachments and notify the sender that you have received it in error. Unintended recipients are prohibited from taking action on the basis of information in this e-mail. Unitek reserves the right, to the extent and under circumstances permitted by applicable law, to review, retain, monitor and intercept e-mail messages to and from its systems. Unitek may also retain and reproduce electronic messages for state, federal, or other regulatory agencies as required by applicable law.

**Comment by dwichers** [comment]
Thinking I had the ideal plugin for a web app I'm building I was disappointed to see this issue when testing it on mobile phones and tablets. I'm writing this hoping that devs will pick this issue op as I don't even have the faintest clue where to start to fix the issue myself.

**Comment by dwichers** [comment]
What an annoying issue! I was so happy with the duallistbox plugin for a web app I'm developing. I might have to use something else even though the plugin still works on mobile devices... The the user experience is just so bad because of this issue.

**Comment by istvan-ujjmeszaros** [comment]
@dwichers which Android version did you try?
I am wondering if it is possible to detect if the browser is Android and use an alternative select element with no style applied on it (hopefully it would work around the issue as it was stated on the Android Public Tracker).

**Comment by dwichers** [comment]
I use Android 13. But the problem also exists on an old iPad (using Safari) with iOS 12.5. It supposes to be an issue that was reported years ago: https://issuetracker.google.com/issues/36928661 but that can't be it. Removing all the styling doesn't fix the issue. I now use the select2 bootstrap plugin instead with some tweaking to keep the options in the order I select them. Because that was the nifty feature I wanted from duallistbox in the first place.
